classes/class-wc-connect-nux.php

```diff
@@ -94,8 +94,10 @@ public function show_pointers( $hook ) {
 			}
 
 			wp_enqueue_style( 'wp-pointer' );
-			wp_enqueue_script( 'wc_services_admin_pointers', plugins_url( 'assets/js/admin-pointers.js', dirname(__FILE__) ), array( 'wp-pointer', 'jquery' ) );
+			$wc_connect_base_url = defined( 'WOOCOMMERCE_CONNECT_DEV_SERVER_URL' ) ? WOOCOMMERCE_CONNECT_DEV_SERVER_URL : plugins_url( 'dist/', __FILE__ );
+			wp_register_script( 'wc_services_admin_pointers', $wc_connect_base_url . 'woocommerce-services-admin-pointers.js', array( 'wp-pointer', 'jquery' ), false, true );
```

Comment:
If you look at `woocommerce-services.php#admin_enqueue_scripts()`, there all the scripts/styles are registered in the same place, but they are only required (`wp_enqueue_script`) when they are actually needed. I would prefer to maintain that pattern, but I don't have a strong opinion about it (the `banner.css` file doesn't follow it already).